README.md

```diff
@@ -0,0 +1,59 @@
+lib-webtopay
+========
+
+The Checkout API (Payment Gateway API) allows for the collection of online payments with many payment methods. 
+The Checkout API is easy to integrate – simply use one of our methods and the checkout processes will be performed 
+automatically. The library can be used to check all the necessary security parameters of transferred and received data.
+More information can be found in documentation: https://developers.paysera.com/en/checkout/basic
```

Comment:
Usually it's best to put links to phrases – no need to see the full URL when reading.

For example,

```
More information can be found in [the documentation](https://developers.paysera.com/en/checkout/basic).

```